packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx

```diff
@@ -829,6 +829,13 @@ const EarnAddLiquidityContainer: React.FC = () => {
     return isFetchingPools || isLoadingCommon;
   }, [isFetchingPools, isLoadingCommon]);
 
+  const hasStakingRewardPool = useMemo(
+    () =>
+      selectPool.poolPath ===
+      "gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000",
```

Comment:
Is `gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000` only pool which has staking reward?

Reply:
https://varmeta.slack.com/archives/C06MXPW7T0A/p1721619673555989?thread_ts=1721617118.020319&cid=C06MXPW7T0A

Reply:
i think i should change to another name